src/mui/detail/Edit.js

```diff
@@ -72,25 +70,18 @@ export class Edit extends Component {
             data,
         });
         const titleElement = data ? <Title title={title} record={data} defaultTitle={defaultTitle} /> : '';
-        // using this.previousKey instead of this.fullRefresh makes
-        // the new form mount, the old form unmount, and the new form update appear in the same frame
-        // so the form doesn't disappear while refreshing
-        const isRefreshing = key !== this.previousKey;
-        this.previousKey = key;
-
         return (
             <div className="edit-page">
-                <Card style={{ opacity: isLoading ? 0.8 : 1 }} key={key}>
```

Comment:
I think you've reintroduced a bug here. Use git blame to see why the `key` was introduced.